.github/workflows/container-image.yml

```diff
@@ -47,6 +47,31 @@ jobs:
       with:
         image: "samba-server:latest"
         retention_days: 1
+  build-server-centos:
+    runs-on: ubuntu-latest
+      #env:
+      #      BUILDAH_FORMAT: oci
```

Comment:
If these are not needed, let's just remove them. I'm not asking you to remove them from existing rules, just avoid added commented out stuff to these new jobs.

Reply:
removed the commented out stuff, @phlogistonjohn